harness/determined/common/api/authentication.py

```diff
@@ -133,12 +143,24 @@ def get_session_token(self, must: bool = True) -> str:
         return self.session.token
 
 
+class AuthContext:
```

Comment:
We already have too many authentication-related objects, can you just return a tuple of (token, user_id) from do_login instead?

Reply:
I asked for this, instead of a dict[str,str], lol. Maybe a namedtuple or regular tuple is best?

Reply:
disadvantage of dict is that it breaks accidentally very easy, whereas I guess a type or tuple both don't.